packages/lambda-xyz/src/__test__/wmts.capability.test.ts

```diff
@@ -145,4 +123,60 @@ o.spec('wmts', () => {
                 'template="https://basemaps.test/v1/tiles/01E7PJFR9AMQFJ05X9G7FQ3XMW/3857/{TileMatrix}/{TileCol}/{TileRow}.png" />',
         );
     });
+
+    o('should support multiple projections', () => {
+        const ts = [new FakeTileSet('aerial', Epsg.Nztm2000), new FakeTileSet('aerial', Epsg.Google)];
+        const xml = new WmtsCapabilities('basemaps.test', Provider, ts);
+        const nodes = xml.toVNode();
+
+        const layers = tags(nodes, 'Layer');
+        o(layers.length).equals(1);
+        const layer = layers[0];
+        const sets = tags(layer, 'TileMatrixSet');
+
+        o(sets.length).equals(2);
+        o(sets[0].toString()).equals('<TileMatrixSet>NZTM2000</TileMatrixSet>');
+        o(sets[1].toString()).equals('<TileMatrixSet>WebMercatorQuad</TileMatrixSet>');
+
+        const tms = tags(nodes, 'TileMatrixSet').filter((f) => f.find('ows:SupportedCRS') != null);
+        o(tms.length).equals(2);
+
+        o(tms[0].find('ows:Identifier')?.textContent).equals('NZTM2000');
+        o(tms[0].find('ows:SupportedCRS')?.textContent).equals('urn:ogc:def:crs:EPSG::2193');
+
+        o(tms[1].find('ows:Identifier')?.textContent).equals('WebMercatorQuad');
+        o(tms[1].find('ows:SupportedCRS')?.textContent).equals('urn:ogc:def:crs:EPSG::3857');
+    });
+
+    o('should support multiple tilesets', () => {
+        const ts = [
+            new FakeTileSet('aerial', Epsg.Nztm2000, 'aerial-title'),
+            new FakeTileSet('01E7PJFR9AMQFJ05X9G7FQ3XMW', Epsg.Nztm2000, 'imagery-title'),
+        ];
+        const nodes = new WmtsCapabilities('basemaps.test', Provider, ts).toVNode();
+        const layers = tags(nodes, 'Layer');
+        o(layers.length).equals(2);
+
+        o(layers[0].find('ows:Title')?.textContent).equals('aerial-title');
+        o(layers[0].find('TileMatrixSet')?.textContent).equals('NZTM2000');
+
+        o(layers[1].find('ows:Title')?.textContent).equals('imagery-title');
+        o(layers[1].find('TileMatrixSet')?.textContent).equals('NZTM2000');
+    });
+
+    o('should support multiple different projections on differnt tiles sets', () => {
```

Comment:
I don't see any tests for the NZTM2000 matrixset (zoom level ID, tile dimension, scale etc)? I think we should check that as well

Reply:
The NZTM2000 matrix set tests are coming in the next PR.